openedx_webhooks/tasks/github.py

```diff
@@ -357,6 +369,25 @@ def get_jira_issue_key(pull_request):
     return None
 
 
+def github_prepare_cherry_pick_comment(github, pull_request, open_edx_release):
+    """
+    Prepares a cherry pick comment used for internal engineers during the window between
+    an Open edX release candidate and the official release.
+
+    * fun_facts includes a piece of trivia in an effort to get an engineer's attention
+    """
+    repos = get_repos_file()
+    if repo in repos:
```

Comment:
`repo` seems to be undefined.  Do we need this test here?  Will this function be called for repos we shouldn't handle?